test/test_user.py

```diff
@@ -97,3 +96,31 @@ def test_patch_username_password_length_requirements(self, helper):
         user.update(username=long_username, password=long_password)
         print("  Then when I then try to login using the newly changed data, it should also be successful")
         helper.login_with_username_password(long_username, long_password)
+
+    def test_admin_list_users(self, helper):
+        print("A super admin user can list all other users")
```

Comment:
Probably a better test is to create a username and check you can actually see user from admin.